sys-devel/mold/mold-9999.ebuild

```diff
@@ -17,7 +17,8 @@ fi
 
 # mold (MIT)
 #  - xxhash (BSD-2)
-LICENSE="MIT BSD-2"
+#  - siphash ( MIT CC0-1.0 )
+LICENSE="MIT BSD-2 CC0-1.0"
```

Comment:
Unless I'm misunderstanding something https://github.com/rui314/siphash is licensed under both MIT and CC0-1.0